cmd/osmosisd/cmd/balances_from_state_export.go

```diff
@@ -315,6 +315,32 @@ Example:
 				snapshotAccs[addr] = account
 			}
 
+			queryClient := authtypes.NewQueryClient(clientCtx)
```

Comment:
@mattverse Is this functionally the same as `app := app.Setup(false)` from my previous commit? Am I using it correctly?

Reply:
lgtm!